core/model/message.go

```diff
@@ -80,3 +80,35 @@ type MessagesStats struct {
 	Read       *int64 `json:"read_count" bson:"read_count"`
 	Unread     *int64 `json:"not_read_count" bson:"not_read_count"`
 }
+
+///
+
+//InputMessage is passed by the adapters for creating a message in the core module
+type InputMessage struct {
+	OrgID string `json:"org_id"`
+	AppID string `json:"app_id"`
+
+	Priority int               `json:"priority"`
+	Subject  string            `json:"subject"`
+	Body     string            `json:"body"`
+	Data     map[string]string `json:"data"`
+
+	//recipients related
+	Recipients               []InputMessageRecipient  `json:"recipients"`
+	RecipientsCriteriaList   []InputRecipientCriteria `json:"recipients_criteria_list"`
+	RecipientAccountCriteria map[string]interface{}   `json:"recipient_account_criteria"`
+	Topic                    *string                  `json:"topic"`
+}
+
+// InputMessageRecipient is passed by the adapters for creating a message in the core module
+type InputMessageRecipient struct {
+	UserID string `json:"user_id"`
+	Name   string `json:"name"`
```

Comment:
Looks like we still have the name in the `InputMessageRecipient` even though it has been removed from `MessageRecipient`. I assume this should be removed as well.

Reply:
Yes! Thanks